tests/test_aligner_bwamem.yml

```diff
@@ -32,49 +32,31 @@
       md5sum: 8569fbdb2c98c6fb16dfa73d8eacb070
     - path: results/reference/bwa/genome.sa
       md5sum: e7cff62b919448a3a3d0fe4aaf427594
-    - path: results/reference/dbsnp/dbsnp_146.hg38.vcf.gz.tbi
```

Comment:
Are we no longer publishing those reference-files? And if so, why is that?

Reply:
yes, but not in this test.
That's part of the new test references in the `igenomes.config` file.
So we're not building it when using this reference, since we provide it